docs/source/routing/observability/telemetry.mdx

```diff
@@ -20,139 +31,300 @@ flowchart LR
+        attributes:
+          otel.name: subgraph
+          operation.name: "subgraph"
+          resource.name:
+            subgraph_operation_name: string
+  exporters:
+    tracing:
+      otlp:
+        enabled: true
+        endpoint: "${env.DATADOG_AGENT_HOST}:4317"
+```
 
-## Router telemetry attributes
+Learn more about [sending traces to DataDog](/graphos/reference/router/telemetry/trace-exporters/datadog) and [trace exporters](/graphos/reference/router/telemetry/trace-exporters/overview) in general.
 
-You can annotate events, metrics, and spans with **attributes**. Attributes are key-value pairs that add contextual information about the router pipeline to telemetry. You can then use these attributes to filter and group data in your APMs.
+## Best practices
 
-Example attributes include:
+### Collect exactly the telemetry you need
 
-* HTTP status code
-* GraphQL operation name
-* Subgraph name
+Effective telemetry provides just the right amount and granularity of information to maintain your graph. Too much data can overwhelm your system, for example, with high cardinality metrics. Too little may not provide enough information to debug issues.
 
-You can use [standard attributes](/router/configuration/telemetry/instrumentation/standard-attributes) or [selectors](/router/configuration/telemetry/instrumentation/selectors) as span attributes.
+Specific events that need to be captured—and the conditions under which they need to be captured—can change as client applications and graphs change. Different environments, such as production and development, can have different observability requirements.
 
-<PremiumFeature>
+Router telemetry is customizable to meet the observability needs of different graphs. Keep in mind your particular environments' and graphs' requirements when configuring your telemetry.
 
-[Custom attributes for spans](/router/configuration/telemetry/instrumentation/spans/#attributes) require a GraphOS [Dedicated or Enterprise plan](https://www.apollographql.com/pricing#observability).
+#### Conditions
 
-</PremiumFeature>
+You can set conditions for instruments and events to only collect telemetry data when necessary. This configuration snippet enables only collecting the configured telemetry data when the `request_header` is equal to "example-value":
 
-## Best practices
+```yaml
+eq:
+  - "example-value"
+  - request_header: x-req-header
+```
+#### Dropping metrics using views
+
+You can use metric exporters' [`view`](/graphos/reference/router/telemetry/metrics-exporters/overview#views) property with the `drop` aggregation to remove certain metrics from being sent to your APM. This configuration snippet removes all instruments that begin with `apollo_router`:
+
+```yaml title="router.yaml"
+telemetry:
+  exporters:
+    metrics:
+      common:
+        service_name: apollo-router
+        views:
+          - name: apollo_router*
+            aggregation: drop
+```
 
 ### Balancing telemetry and router performance
```

Comment:
Comment from @devfreddy-apollo on this section:

> The router team likely has some actual stats on this impact, but it's not something I've seen with customers at all. The router is performant enough to add A LOT of telemetry before this is an issue. It's good to document, but I'm not sure it needs to be prioritized in an opening primer. That might be my personal opinion, though.

This section was originally written by the router team, so I'm inclined to keep it. @BrynCooke would you be able to comment on whether this is worth calling out upfront?